ImageSegmentation/ImageSegmentation/TorchModule.mm

```diff
@@ -0,0 +1,259 @@
+// Copyright (c) 2020 Facebook, Inc. and its affiliates.
+// All rights reserved.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+
+
+
+#import "TorchModule.h"
+#import <LibTorch/LibTorch.h>
+#import <CoreImage/CoreImage.h>
+#import <ImageIO/ImageIO.h>
+#import <UIKit/UIKit.h>
+
+@implementation TorchModule {
+ @protected
+  torch::jit::script::Module _impl;
+}
+
+UIImageView* m_imageView;
```

Comment:
The global value doesn't make sense here. Can we move this to the UI layer via @property(...) or ivar `_imageView`?

Reply:
Sure and thanks.. it was meant for a quick hack to see the result showing.